scipy/stats/stats.py

```diff
@@ -3086,10 +3086,33 @@ def pearsonr(x, y):
     p-value : float
         2-tailed p-value
 
+    Notes
+    -----
+
+    The correlation coefficient is calculated as follows:
+
+    .. math::
+
+        r_{pb} = \frac{\sum (x - m_x) (y - m_y)
+                       }{\sqrt{\sum (x - m_x)^2 (y - m_y)^2}}
+
+    where :math:`m_x` is the mean of the vector :math:`x` and :math:`m_y` is
+    the mean of the vector :math:`y`.
+
+
     References
     ----------
     http://www.statsoft.com/textbook/glosp.html#Pearson%20Correlation
 
+    Examples
+    --------
+    >>> a = np.array([0, 0, 0, 1, 1, 1, 1])
```

Comment:
This needs `from scipy import stats` --- only `import numpy as np` is assumed in the examples